tests/Feature/Modules/Crm/TitleCrudTest.php

```
@@ -0,0 +1,117 @@
+        /* Arrange */
+        $visibleTitle = Title::factory()->create();
+        $otherTitle   = Title::factory()->create();
+
+        /* Act */
+        $component = Livewire::actingAs($this->user)
+            ->test(ManageTitles::class);
+
+        /* Assert */
+        $component->assertCanSeeTableRecords([$visibleTitle])
+            ->assertCanSeeTableRecords([$otherTitle]);
+    }
```

PHPMD (2.15.0)

The method it_lists_titles is not named in camelCase. (undefined)

(CamelCaseMethodName)